karma.conf.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* eslint-disable */
```

Comment:
We should prefer to fix any lint errors, rather than disabling linting.

Reply:
I assumed this, being outside `src/` would be exempt. fixed

Reply:
Ah, I think the eslint command in package.json isn't picking it up. I'll fix this.

Reply:
in f8b9712 I modified it slightly so the linter stopped complaining (at least on mine)